pep-0001.txt

```diff
@@ -411,23 +411,13 @@ Each PEP should have the following parts:
 2. Abstract -- a short (~200 word) description of the technical issue
    being addressed.
 
-3. Copyright/public domain -- Each PEP must either be explicitly
-   labeled as placed in the public domain (see this PEP as an
-   example) or licensed under the `Open Publication License`_.
-
-4. Specification -- The technical specification should describe the
-   syntax and semantics of any new language feature.  The
-   specification should be detailed enough to allow competing,
-   interoperable implementations for at least the current major Python
-   platforms (CPython, Jython, IronPython, PyPy).
-
-5. Motivation -- The motivation is critical for PEPs that want to
+3. Motivation -- The motivation is critical for PEPs that want to
    change the Python language.  It should clearly explain why the
    existing language specification is inadequate to address the
    problem that the PEP solves.  PEP submissions without sufficient
    motivation may be rejected outright.
 
-6. Rationale -- The rationale fleshes out the specification by
+4. Rationale -- The rationale fleshes out the specification by
    describing what motivated the design and why particular design
```

Comment:
Maybe drop "what motivated the design and"? Since that should already be in the Motivation section.